e2e-tests/build-and-run

```diff
@@ -7,8 +7,8 @@ test_dir="$(dirname $0)"
 src_dir="$(realpath $test_dir/..)"
 
 if [[ $BUILD != "0" ]]; then
-    echo "building, set BUILD=0 to suppress this"
-    $test_dir/build
```

Comment:
afaik we are using spaces everywhere, pls revert

Reply:
done